packages/devtools_app/lib/src/screens/network/network_request_inspector_views.dart

```diff
@@ -348,7 +340,15 @@ class HttpTextResponseViewer extends StatelessWidget {
         }
 
         return switch (currentLocalResponseType) {
-          NetworkResponseViewType.json => JsonViewer(encodedJson: responseBody),
+          NetworkResponseViewType.json =>
+            // Don't load the JsonViewer if we can't decode the JSON.
+            _isJsonDecodable(responseBody)
+                ? JsonViewer(encodedJson: responseBody)
+                : TextViewer(
+                  // We could also include the decoding exception.
+                  text: responseBody,
+                  style: textStyle.copyWith(color: Colors.red),
```

Comment:
@kenzieschmoll do we want to make this red?

Reply:
I think we should leave the default text color and remove this style. 

@srawlins for future reference, we try to always use a color from the inherited color scheme `Theme.of(context).colorScheme.foo`. You can see the color schemes for devtools in `theme.dart`.

Reply:
SG. Users will ask why the JsonViewer is not loading with a JSON request. Is there any other signal we can show in the UI or leave as is?

Reply:
You could push an error notification: `notificationService.pushError`. I would make it not reportable and ensure it dismisses on a timeout

Reply:
That might push a lot of notifications, haha. Like if several responses per second come back as unparsable.

I've removed the red for now and left a comment.